### ISSUE TITLE
webpack failed to compile for Vue.js monaca preview

### ISSUE THREAD

**Issue by Youdaman** [monaca/monaca-cli#83]
Steps to repro:

1. monaca create test
2. choose "Onsen UI and Vue.js"
3. choose any of the templates
4. cd test
5. monaca preview (or any of the commands)

Expected:

See a preview of the app.

Actual:

Webpack failed to compile, here's the output:
```
$ monaca preview
Build completed in 7.252s

chunk    {0} bundle.js, main.css (main) 4.19 MB [rendered]

ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
 @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:60-102

ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./img/android-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
 @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:502-548

ERROR in ./~/onsenui/css-components-src/src/onsen-css-components.css
Module build failed: ModuleNotFoundError: Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
    at C:\Users\Youdaman\.cordova\node_modules\webpack\lib\Compilation.js:229:38
    at onDoneResolving (C:\Users\Youdaman\.cordova\node_modules\webpack\lib\NormalModuleFactory.js:29:20)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\lib\NormalModuleFactory.js:85:20
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:726:13
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:52:16
    at done (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:241:17)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:44:16
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:723:17
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\async\lib\async.js:167:37
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:24:19
    at onResolved (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:38:18)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:127:10
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:191:15
    at applyPluginsParallelBailResult.createInnerCallback.log (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:110:4)
    at loggingCallbackWrapper (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\createInnerCallback.js:21:19)
    at C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\tapable\lib\Tapable.js:134:6
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\DirectoryDefaultFilePlugin.js:21:12)
    at Storage.provide (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:52:20)
    at CachedInputFileSystem.stat (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:132:20)
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\DirectoryDefaultFilePlugin.js:18:6)
    at Tapable.applyPluginsParallelBailResult (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\tapable\lib\Tapable.js:139:14)
    at Tapable.<anonymous> (C:\Users\Youdaman\.cordova\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:103:8)
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 541 kB [rendered]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 405 kB [rendered]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 386 kB [rendered]

    ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
    Module not found: Error: Cannot resolve 'file' or 'directory' ./img/ios-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
     @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:60-102

    ERROR in C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css
    Module not found: Error: Cannot resolve 'file' or 'directory' ./img/android-search-input-icon.svg in C:\Users\Youdaman\Desktop\test\node_modules\onsenui\css-components-src\src\components
     @ C:/Users/Youdaman/.cordova/~/css-loader?sourceMap!./~/onsenui/css-components-src/src/components/search-input.css 6:502-548
webpack: Failed to compile.
```

**Comment by Youdaman** [comment]
Seems to be a problem with cordova looking at the above error messages, so I'm currently blowing away my npm modules (including cordova), updating Node and re-doing npm install -g monaca to see if I can resolve the issues.

**Comment by Youdaman** [comment]
Nope, webpack build still fails, will investigate to see if anyone else is having the issue with the cordova module and webpack.

**Comment by Youdaman** [comment]
Another thing to note was the errors are to do with svg files. Will compare Vue.js with Angular/React monaca templates to see if they have the same problem, or if not, what they do differently in their webpack configs (assuming that's the problem)

**Comment by Youdaman** [comment]
Angular template doesn't use webpack, but React does and works fine, so looking into differences with Vue.js template

**Comment by Youdaman** [comment]
Seems the Vue.js templates can't be fetched, hangs at "Downloading template..."
```
$ monaca create test-vue
? Choose a category: Onsen UI and Vue.js
? Select a template - Press P to see a preview onsenui-v2-vue-minimum

Downloading template...
```

**Comment by Youdaman** [comment]
Found the problem. The file `node_modules\onsenui\css-components-src\src\components\search-input.css` tries to load some svg images that don't exist.

Solution: copy the same file from the same location within a React template project over to your Vue template project and then `monaca preview` etc just works.

Bottom line, the Vue.js template for the Monaca CLI is broken due to a bad/broken search-input.css file. Please fix (by replacing the same file from the React template which works fine).

**Comment by asial-matagawa** [comment]
@Youdaman 
Thank you for reporting & investigating!
@Youdaman 
Thank you for reporting!
I fixed this issue in monaca/monaca-lib#105.

Closing this.

**Comment by skapin** [comment]
Thank you a lot @Youdaman
